include/mbedtls/pk_info.h

```diff
@@ -71,6 +71,13 @@ extern "C" {
  * \note If you are using the PK interface to perform operations on
  * keys, call the functions in pk.h. The interface in this file should only
  * be used by implementers of opaque key engines.
+ *
+ * \warning: Do not declare this structure directly! It may be extended in
+ * future* versions of Mbed TLS. Call the macro
+ * MBEDTLS_PK_OPAQUE_INFO_1() or MBEDTLS_PK_OPAQUE_INFO_ASYNC_1() instead.
```

Comment:
Question: what's this `ASYNC` macro? I'm not seeing it in this commit. How does an implementer know whether to use one or the other?

Reply:
It has been removed when the work has been split to remove async changes. Good that you've spotted that.